### ISSUE TITLE
Is it normal that composer downloads this outside of vendor? 

### ISSUE THREAD

**Issue by ndobromirov** [hechoendrupal/drupal-console-en#258]
This package was added as a dependency from `drupal/console-core` and everything that is managed by composer lands correctly in the vendor folder, except this one...

**Comment by guardiola86** [comment]
I don't think it is. Currently I can't update packages in composer because of this.

**Comment by ndobromirov** [comment]
I thought so...
Should we fix it :D. For now I have this worked-around with git-ignore, but it should be fixed so it knows where to download the package (**vendor**).
